Workbooks/CyberArkEPV.json

```diff
@@ -0,0 +1,155 @@
+{
```

Comment:
I tested this workbook and it did not render - you can copy paste the json of the workbook and go to https://portal.azure.com -> Azure Sentinel -> Workbooks -> New workbook -> Edit -> Click on code which is the '<>' icon. Then paste the workbook content completely- the workbook as-is won't render and errors out because of the metadata info that's pasted at the end of this workbook. Please delete that and then it renders fine. Please fix and test this before re-submitting.